### PR TITLE
Quick Pad 5.2.0

### PR DESCRIPTION
**Pull request by yair100**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/5508)

### CODE REVIEW

**Comment by KevinLaMS** [comment]
image

**Comment by yaichenbaum** [comment]
@KevinLaMS Being that WinGet will support installing apps from the store, I don't think it's worth investing so much for this package right now. Is there a process to remove an app from WinGet or do I just need to delete the folder for it?

**Comment by KevinLaMS** [comment]
I think what we should do is keep this PR active as active so that once dependencies are done, we can add this if we want.
I agree, the 404 error we get with the current package is not good.  It would be great if you want to put through the RM for that package.

**Comment by yaichenbaum** [comment]
> I think what we should do is keep this PR active as active so that once dependencies are done, we can add this if we want.
> I agree, the 404 error we get with the current package is not good. It would be great if you want to put through the RM for that package.

I removed the outdated package for Quick Pad over here #5627.

**Comment by vedantmgoyal2009** [comment]
@denelon @KevinLaMS This PR can be closed as the version of the manifest being used is 0.1.0.

**Comment by yaichenbaum** [comment]
> @denelon @KevinLaMS This PR can be closed as the version of the manifest being used is 0.1.0.

How can that be resolved?

File: manifests/QuickPad/QuickPad/5.2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+License: GNU
+LicenseUrl: https://github.com/yaichenbaum/Quick-Pad/blob/master/LICENSE.md
+AppMoniker: Quick Pad
+Description: Quick Pad is a modern notepad app featuring fluent design
+Homepage: https://github.com/yaichenbaum/Quick-Pad
+Installers:
+  - Arch: Neutral
+    Url: https://github.com/yaichenbaum/Quick-Pad/releases/download/5.2.0/5.2.0.0.Msixbundle
+    Sha256: 0313FA5C1A1B82F95ECF7BAFAB8E4E066D246301FE1846377FD34607282739DC
+    InstallerType: msix
```
Comment (vedantmgoyal2009) [inline manifests/QuickPad/QuickPad/5.2.0.yaml:14]:
```suggestion
PackageIdentifier: QuickPad.QuickPad
PackageVersion: 5.2.0
PackageName: Quick Pad
Publisher: Quick Pad
License: GNU
LicenseUrl: https://github.com/yaichenbaum/Quick-Pad/blob/master/LICENSE.md
Moniker: quickpad
ShortDescription: Quick Pad is a modern notepad app featuring fluent design.
PackageUrl: https://github.com/yaichenbaum/Quick-Pad
Installers:
- Architecture: neutral
  InstallerUrl: https://github.com/yaichenbaum/Quick-Pad/releases/download/5.2.0/5.2.0.0.Msixbundle
  InstallerSha256: 0313FA5C1A1B82F95ECF7BAFAB8E4E066D246301FE1846377FD34607282739DC
  InstallerType: msix
PackageLocale: en-US
ManifestType: singleton
ManifestVersion: 1.0.0
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
@yaichenbaum Sorry, I didn't tell earlier but the path of the manifest should be `manifests/QuickPad/QuickPad/5.2.0/QuickPad.QuickPad.yaml`

**Comment by yaichenbaum** [comment]
Still working on this.

**Comment by yaichenbaum** [comment]
@vedantmgoyal2009 I'm not sure why this isn't working, any ideas?

**Comment by quhxl** [comment]
The folder should be `manifests\q\QuickPad\QuickPad\5.2.0` and not `manifests\QuickPad\QuickPad\5.2.0`

**Comment by vedantmgoyal2009** [comment]
https://github.com/microsoft/winget-pkgs/pull/5508#issuecomment-916959451

**Comment by arjun-msft** [comment]
@yaichenbaum this appears to be a paid app, after I install it and run it, it is blocked and takes me to the store. We are unable to install paid apps at the moment, I believe we are blocked on this: https://github.com/microsoft/winget-cli/issues/908.

**Pull request closed without merging**